opendevin/runtime/browser/browser_env.py

```diff
@@ -47,6 +47,16 @@ def __init__(
         self.process = multiprocessing.Process(
             target=self.browser_process,
         )
+
+        try:
+            self.original_cwd = os.getcwd()
+        except FileNotFoundError:
```

Comment:
I'm just curious under what scenario this will happen... Under WSL? I never encounter such issue myself though.

Reply:
+1. And hopefully it will go away once we containerize browser environment